### ISSUE TITLE
CVE-2020-8285 and CVE-2020-8286 (curl)

### ISSUE THREAD

**Issue by supersven** [nginxinc/docker-nginx#481]
Affected: `nginx:1.19.5-alpine` (at least)
Found with klar/clair in a GitLab CI pipeline.

```
[
  {
    "fixedby": "7.69.1-r3",
    "severity": "Unknown",
    "link": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8285",
    "description": "",
    "namespace": "alpine:v3.12",
    "vulnerability": "CVE-2020-8285",
    "featureversion": "7.69.1-r2",
    "featurename": "curl"
  },
  {
    "fixedby": "7.69.1-r3",
    "severity": "Unknown",
    "link": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8286",
    "description": "",
    "namespace": "alpine:v3.12",
    "vulnerability": "CVE-2020-8286",
    "featureversion": "7.69.1-r2",
    "featurename": "curl"
  }
]
```

- https://curl.se/docs/CVE-2020-8285.html
- https://curl.se/docs/CVE-2020-8286.html

**Comment by supersven** [comment]
This is only a matter of updating the packages. E.g. this `Dockerfile` solves these CVEs:

```dockerfile
FROM nginx:1.19.5-alpine

# Update curl - https://github.com/nginxinc/docker-nginx/issues/481
RUN apk update && apk upgrade -U curl --no-cache
```

**Comment by thresheek** [comment]
Fixed in 1.19.6-alpine.
